ENGAGEHF/Dashboard/ToDo/ToDoItem.swift

```diff
@@ -0,0 +1,21 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+
+
+struct ToDoItem: Identifiable {
+    let id: String
+    var task: String
```

Comment:
I think we have a fixed set of TODOs and if they are only stored locally we could use an enum to e.g. define a clear set of TODOs and we rather display them (could have a computed property that presents a localized description).